app/javascript/orderscreen/orderhistory.vue

```diff
@@ -1,227 +0,0 @@
-<template lang="html">
```

Comment:
This component has been split up into `components/orderhistory.vue` and `components/activityorders.vue`